front/lib/solutions/transcripts/temporal/worker.ts

```diff
@@ -0,0 +1,31 @@
+import type { Context } from "@temporalio/activity";
```

Comment:
let's move this one directory down so that we have one queue for all solutions stuff